README.md

```diff
@@ -0,0 +1,17 @@
+# LAP2 24 hour Code Challlenge
+This is our lap 2 24 hour code challenge which covers HTML/CSS/JS client and an Express server connected to PostgreSQL database.
+
+# Installion and Usage
+- Clone or downlaod the repo.
+- Open terminal and navigate to the _scripts folder.
+- run startDev.sh to launch server.
+- run stop.sh to stop server.
+
```

Comment:
Could add an implementation section, telling the user what you implementated and hwo you did it